package.json

```diff
@@ -53,7 +53,10 @@
   "devDependencies": {
     "autoprefixer": "^10.4.7",
     "eslint": "^8.16.0",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
```

Comment:
This is for html syntax

Reply:
For documentation
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y

One of the rules
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/112261cbc84f5b7d74de9b427b529a10b41faece/docs/rules/alt-text.md